src/roles/nuh-deploy/templates/config.yml.j2

```diff
@@ -1,3 +1,10 @@
+geo: 
+{% if vsd_sa_or_ha is match('ha') %}
+  active: true
+{% else %}
+  active: false
```

Comment:
If this is not a GEO diverse deployment, and I doubt it is, then active should always be true.